test/uk/gov/hmrc/gform/gformbackend/DemoApp.scala

```diff
@@ -39,9 +39,9 @@ object DemoApp extends App {
   val userId = UserId("TestUser")
 
   val result = for {
-    formId <- gformConnector.newForm(formTemplateId, userId, None)
-    form   <- gformConnector.getForm(formId)
-    _ = println(s"formId = $formId")
+    newFormData <- gformConnector.newForm(formTemplateId, userId, None)
+    form        <- gformConnector.getForm(newFormData.formId)
+    _ = println(s"formId = ${newFormData.formId}")
```

Comment:
Are you deliberately leaving these printlns in?

Reply:
I'm not sure what purpose of `DemoApp` is. We probably don't need it, but since println was there before I simply do not want to remove it, in case someone relies on it 🤷‍♂.